cmd/nginx-ingress/main.go

```diff
@@ -154,6 +154,9 @@ var (
 	enablePrometheusMetrics = flag.Bool("enable-prometheus-metrics", false,
 		"Enable exposing NGINX or NGINX Plus metrics in the Prometheus format")
 
+	prometheusTLSSecretName = flag.String("prometheus-tls-secret", "",
+		`A Secret with a TLS certificate and key for TLS termination of the prometheus endpoint.`)
```

Comment:
neat: for consistent naming. we used Prometheus before. could you capitalize other new ocurrences?
```suggestion
		`A Secret with a TLS certificate and key for TLS termination of the Prometheus endpoint.`)
```

Reply:
I think it's fine as it is.